Extension/package.json

```diff
@@ -42,6 +42,168 @@
+            "title": "Build your C++ file",
+            "description": "Create a default build task for your project. \n[Learn more](https://code.visualstudio.com/docs/cpp/config-msvc#_build-helloworldcpp)",
+            "when": "isWindows",
+            "media": {
+              "altText": "markdown file with instructions for configuring default build task on MSVC with Windows",
+              "path": "walkthrough/build-project-windows.md"
+            }
+          },
+          {
+            "id": "run.project.mac",
+            "title": "Run and debug your C++ file",
+            "description": "Create a debug configuration for your project. \n[Learn more](https://code.visualstudio.com/docs/cpp/config-clang-mac#_debug-helloworldcpp)",
+            "when": "isMac",
+            "media": {
+              "altText": "markdown file with instructions for starting a debugging session on macOS",
+              "path": "walkthrough/run-and-debug-project-mac.md"
+            }
+          },
+          {
+            "id": "run.project.linux",
+            "title": "Run and debug your C++ file",
+            "description": "Create a debug configuration for your project. \n[Learn more](https://code.visualstudio.com/docs/cpp/config-linux#_debug-helloworldcpp)",
+            "when": "isLinux",
+            "media": {
+              "altText": "markdown file with instructions for starting a debugging session on Linux",
+              "path": "walkthrough/run-and-debug-project-linux.md"
+            }
+          },
+          {
+            "id": "run.project.windows",
+            "title": "Run and debug your C++ file",
+            "description": "Create a debug configuration for your project. \n[Learn more](https://code.visualstudio.com/docs/cpp/config-msvc#_debug-helloworldcpp)",
+            "when": "isWindows",
+            "media": {
+              "altText": "markdown file with instructions for starting a debugging session on Windows",
+              "path": "walkthrough/run-and-debug-project-windows.md"
+            }
+          },
+          {
+            "id": "install.cmake.tools",
+            "title": "Install CMake Tools",
+            "description": "Do you build your project with CMake? Install the CMake Tools extension to seamlessly build and debug CMake project. \n[Install CMake Tools](command:workbench.extensions.installExtension?%22ms-vscode.cmake-tools%22)",
+            "doneOn": {
+              "command": "workbench.extensions.installExtension"
+            },
+            "media": {
+              "altText": "screenshot of CMake Tools extension page",
+              "path": {
+                "light": "walkthrough/cmake-tools-light.png",
+                "dark": "walkthrough/cmake-tools-dark.png",
+                "hc": "walkthrough/cmake-tools-hc.png"
+              }
+            }
+          },
+          {
+            "id": "watch.tutorials",
+            "title": "Lean back and get started",
```

Comment:
That's a good point. This wording was copied from one of VS Code's walkthroughs, and they've seen high success with this task / wording. And since we're only shipping an English version to start, I think we should keep it for now. But I'll ask the VS Code team if they've considered this point. I wonder if the localization team would take care of this when they localize our content...

Reply:
Ok